sat/cli/bootsys/mgmt_power.py

```diff
@@ -425,10 +425,19 @@ def do_power_on_ncns(args):
                     if ncn_group == included_ncn_groups['storage']:
                         try:
                             do_ceph_unfreeze(included_ncn_groups)
+                            LOGGER.info('Ceph unfreeze completed successfully on storage NCNs.')
+
                         except FatalPlatformError as err:
                             LOGGER.error(f'Failed to unfreeze Ceph on storage NCNs: {err}')
-                            sys.exit(1)
-                        LOGGER.info('Ceph unfreeze completed successfully on storage NCNs.')
+                            # Use pester_choices to prompt the user
+                            user_choice = pester_choices("Ceph is not healthy. Do you want to continue anyway?",
```

Comment:
Very minor, but there is some inconsistent use of single vs. double quotes:

```suggestion
                            user_choice = pester_choices('Ceph is not healthy. Do you want to continue anyway?',
```

Reply:
addressed them